package.json

```diff
@@ -11,7 +11,7 @@
     "dist"
   ],
   "scripts": {
-    "start": "rm -rf dist && tsc-watch --onSuccess \"node ./dist/index.js\" ",
```

Comment:
@ddesrosier  I'm curious why on build success we run `node ./dist/index.js` ?

Reply:
I don't know... Probably just coming from a blog and I didn't question it

Reply:
That's the answer I was hoping for 😄